openedx_events/content_authoring/signals.py

```diff
@@ -198,3 +199,14 @@
         "library_block": LibraryBlockData,
     }
 )
+
+# .. event_type: org.openedx.content_authoring.object.tagged.v1
```

Comment:
If the name is CONTENT_OBJECT_TAGGED then this should be: `org.openedx.content_authoring.content.object.tagged.v1`. Also, should this be called `CONTENT_OBJECT_TAGS_CHANGEG` instead?

Reply:
Yes, that makes more sense, I've updated it.